src/eval-frame/actions/actions.js

```diff
@@ -1,17 +1,10 @@
-import messagePasserEval from "../../shared/utils/redux-to-port-message-passer";
 import {
   addConsoleEntryInEditor,
   sendStatusResponseToEditor
 } from "./editor-message-senders";
 
 import { IODIDE_EVALUATION_RESULTS } from "../iodide-evaluation-results";
 
-const CodeMirror = require("codemirror"); // eslint-disable-line
```

Comment:
this is all just dead code that has not been needed in the eval frame for a long time